lib/daru/io/io.rb

```diff
@@ -275,6 +275,17 @@ def html_table_to_dataframe(table)
           order: table[:order],
           name: table[:name]
       end
+
+      CSV::Converters[:boolean_converter] = lambda do |f, _|
```

Comment:
Obviously people can still define their own custom boolean converters but I think it's good to have some sort of general convenience method built in. (For example Pandas converts booleans by default and I was surprised there was no built-in method here as well).